src/OpenTelemetry.AutoInstrumentation/Instrumentations/Kafka/MessagingAttributes.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="MessagingAttributes.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.AutoInstrumentation.Instrumentations.Kafka;
+
+internal static class MessagingAttributes
```

Comment:
It will be good to add a comment with the semantic convention version being used here, a link to the respective tag (or sha in lack of that) is desirable.

Reply:
added in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3055/commits/f528de7b9da450b46677ebc8cb41c011ea86548c